change/@fluentui-eslint-plugin-0ad87b56-ee4e-44aa-a38d-efa4d79371d7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32980&commitId=f4ca73e7cd12046341cb93bd9ddee82f5c260d88&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>